### ISSUE TITLE
Bundle missing from history

### ISSUE THREAD

**Issue by ghost** [iotaledger/hornet#645]
**Describe the bug**
I have a node that has been running for a few weeks with short interruptions.
Currently the pruning delay is set to 258000.
I make a `getNodeInfo` call to get the `milestoneStartIndex` which is 1559930. This milestone was on 22nd August.
The following transaction was made yesterday: GBFYYHQUFJPJAQKHIVHIZFNBVBVMHUYDWT9SNGLZIGKNVFKCWR9ZDQRILADTWLYJLXSFU9KJNNTXESTUW and it cannot be found on my node:
```
curl -X POST -H 'Content-Type: application/json' -H 'X-IOTA-API-Version: 1' 'http://<address>' --data '{"command": "findTransactions","bundles":["GBFYYHQUFJPJAQKHIVHIZFNBVBVMHUYDWT9SNGLZIGKNVFKCWR9ZDQRILADTWLYJLXSFU9KJNNTXESTUW"]}'
{"hashes":[],"duration":0}
```
I did increase the pruning delay from 40000 to 258000 yesterday before the transaction was submitted.

**Expected behavior**
The response should include the hashes.

**Environment information:**
 - RAM: 8 GB
 - Cores: 16
 - HORNET version: 0.5.1 (updated today from 0.5.0)

**Additional context**
No error in the logs. The node is currently synchronised:
```
...
req(qu/pe/proc/lat): 00000/00000/00000/0000ms, reqQMs: 0, processor: 00000, LSMI/LMI: 1615680/1615680, TPS (in/new/out): 00042/00017/00102, Tips (non-/semi-lazy): 108/0
req(qu/pe/proc/lat): 00000/00000/00000/0000ms, reqQMs: 0, processor: 00000, LSMI/LMI: 1615680/1615680, TPS (in/new/out): 00033/00012/00068, Tips (non-/semi-lazy): 110/0
req(qu/pe/proc/lat): 00000/00000/00000/0000ms, reqQMs: 0, processor: 00000, LSMI/LMI: 1615680/1615680, TPS (in/new/out): 00079/00020/00991, Tips (non-/semi-lazy): 104/0
req(qu/pe/proc/lat): 00000/00000/00000/0000ms, reqQMs: 0, processor: 00000, LSMI/LMI: 1615680/1615680, TPS (in/new/out): 00079/00020/00991, Tips (non-/semi-lazy): 104/0
req(qu/pe/proc/lat): 00000/00000/00000/0000ms, reqQMs: 0, processor: 00000, LSMI/LMI: 1615680/1615680, TPS (in/new/out): 00056/00014/00554, Tips (non-/semi-lazy): 104/0
req(qu/pe/proc/lat): 00000/00000/00000/0000ms, reqQMs: 0, processor: 00000, LSMI/LMI: 1615680/1615680, TPS (in/new/out): 00057/00022/00127, Tips (non-/semi-lazy): 108/0
req(qu/pe/proc/lat): 00000/00000/00000/0000ms, reqQMs: 0, processor: 00000, LSMI/LMI: 1615680/1615680, TPS (in/new/out): 00057/00022/00127, Tips (non-/semi-lazy): 107/0
2020-08-28T13:25:01Z    INFO    Tangle  Valid milestone detected! Index: 1615681, Hash: AILEKEHKWPL9GJHQLQDTYDQGJF9JNOXSN9MJMAYAWOJ9VAXQBVIDHORVHZOVKXYZULNLR9INQLUQ99999
2020-08-28T13:25:01Z    INFO    Tangle  Run solidity check for Milestone (1615681)...
2020-08-28T13:25:01Z    INFO    Tangle  Solidifier finished: txs: 3, collect: 0s, solidity 0s, propagation: 1ms, total: 1ms
2020-08-28T13:25:01Z    INFO    Tangle  Milestone confirmed (1615681): txsConfirmed: 142, txsValue: 0, txsZeroValue: 142, txsConflicting: 0, collect: 8ms, total: 32ms
2020-08-28T13:25:01Z    INFO    Tangle  New solid milestone: 1615681, 16.40 TPS, 14.20 CTPS, 86.59% conf.rate
req(qu/pe/proc/lat): 00000/00000/00000/0000ms, reqQMs: 0, processor: 00000, LSMI/LMI: 1615681/1615681, TPS (in/new/out): 00065/00023/00134, Tips (non-/semi-lazy): 103/0
...
```

config.json:
```

{
--
"useProfile": "auto",
"httpAPI": {
"basicAuth": {
"enabled": false,
"username": "",
"passwordHash": "",
"passwordSalt": ""
},
"excludeHealthCheckFromAuth": false,
"permitRemoteAccess": [
"attachToTangle",
"getNodeInfo",
"getBalances",
"checkConsistency",
"getTipInfo",
"getTransactionsToApprove",
"getInclusionStates",
"getNodeAPIConfiguration",
"wereAddressesSpentFrom",
"broadcastTransactions",
"findTransactions",
"storeTransactions",
"getTrytes"
],
"whitelistedAddresses": [],
"bindAddress": "0.0.0.0:9999",
"limits": {
"bodyLengthBytes": 1000000,
"findTransactions": 1000,
"getTrytes": 1000,
"requestsList": 1000
}
},
"tipsel": {
"belowMaxDepth": 15
},
"db": {
"path": "/data/hornet/mainnetdb",
"debug": false
},
"logger": {
"level": "info",
"disableCaller": true,
"encoding": "console",
"outputPaths": [
"stdout"
]
},
"snapshots": {
"loadType": "local",
"local": {
"intervalSynced": 50,
"intervalUnsynced": 1000,
"path": "snapshots/mainnet/export.bin",
"downloadURLs": [
"https://ls.manapotion.io/export.bin",
"https://x-vps.com/export.bin",
"https://dbfiles.iota.org/mainnet/hornet/latest-export.bin"
]
},
"global": {
"path": "snapshotMainnet.txt",
"spentAddressesPaths": [
"previousEpochsSpentAddresses1.txt",
"previousEpochsSpentAddresses2.txt",
"previousEpochsSpentAddresses3.txt"
],
"index": 1050000
},
"pruning": {
"enabled": true,
"delay": 258000
}
},
"spentAddresses": {
"enabled": true
},
"network": {
"preferIPv6": false,
"gossip": {
"bindAddress": "0.0.0.0:15600",
"reconnectAttemptIntervalSeconds": 60
},
"autopeering": {
"bindAddress": "0.0.0.0:14626",
"runAsEntryNode": false,
"entryNodes": [
"FvfwJuCMoWJvcJLSYww7whPxouZ9WFJ55uyxTxKxJ1ez@enter.hornet.zone:14626",
"EkSLZ4uvSTED1x6KaGzqxoGxjbytt2rPVfbJk1LRLCGL@enter.manapotion.io:18626",
"iotaMk9Rg8wWo1DDeG7fwV9iJ41hvkwFX8w6MyTQgDu@enter.thetangle.org:14627",
"12w9FrzMdDQ42aBgFrv1siHuJMhuZ4SMVHRFSS7Zb72W@entrynode.iotatoken.nl:14626",
"DboTc1v61Xdyvggj8VRszy92ScUTLgfwZaHvXsU8zr7e@entrynode.tanglebay.org:14626",
"31Tz9meznQMm7qSDUgyMmYVeHUCGA7za5Suvbom5hpE9@bender.iota.autopeering.com:14626"
],
"seed": ""
}
},
"node": {
"alias": "",
"showAliasInGetNodeInfo": false,
"disablePlugins": [
"WarpSync",
"Dashboard",
"Graph",
"MQTT",
"Prometheus",
"Spammer",
"ZMQ"
],
"enablePlugins": []
},
"profiling": {
"bindAddress": "localhost:6060"
}
}
```

**Comment by muXxer** [comment]
Can you find it in the dashboard of the node?? I found a bug in `findTransactions`.
It will be fixed in 0.5.2

**Comment by ghost** [comment]
hi @muXxer , I cannot verify that - I do not use the dashboard plugin:
```
"disablePlugins": [
"WarpSync",
"Dashboard",
"Graph",
"MQTT",
"Prometheus",
"Spammer",
"ZMQ"
],
```

**Comment by muXxer** [comment]
Fix in #649 
pls update
